pyvo/mivot/__init__.py

```diff
@@ -0,0 +1,9 @@
+import os
+from pyvo.mivot.utils.logger_setup import LoggerSetup
+from pyvo.utils import activate_features
+
+activate_features('MIVOT')
+
+logger = LoggerSetup.get_logger()
+LoggerSetup.set_debug_level()
```

Comment:
Since this is a global setting, would it be better to leave setting this to the user?  Of course they can call `LoggerSetup.set_default_format()` to revert this, but they might be more aware of that feature if they had called `LoggerSetup.set_debug_level()` to begin with.

Reply:
The mivot-specific logger has been removed (6bc33b73a3e298a3685bb536be217eb8bf174ad5). The default Python logger is used in this package now (a little less code in the PR).